core/classifiers/base.py

```diff
@@ -84,11 +93,12 @@ def train(self, training_data):
         raise NotImplementedError
 
     @abc.abstractmethod
-    def validate(self, classifier_data):
-        """Validates classifier data.
+    def validate(self, frozen_model_proto):
+        """Validates frozen model protobuf object containing parameters of
```

Comment:
Nit on wording: ``Validates the specified frozen model protobuf...``

Reply:
Done